src/Infrastructure/Array/tests/makefile

```diff
@@ -18,7 +19,7 @@ TESTS_BUILD   = $(ESMF_TESTDIR)/ESMF_ArrayCreateGetUTest \
                 $(ESMF_TESTDIR)/ESMF_ArrayRedistUTest \
                 $(ESMF_TESTDIR)/ESMF_ArrayRedistPerfUTest \
                 $(ESMF_TESTDIR)/ESMF_ArrayHaloUTest \
-                $(ESMF_TESTDIR)/ESMC_ArrayUTest 
```

Comment:
Probably exclusive white space change.

Reply:
Is there a git configuration to fix this?

Reply:
From what I can tell just best to revert...nvm.